src/ocrd/processor/base.py

```diff
@@ -351,18 +360,21 @@ def process_page_file(self, *input_files : OcrdFile) -> None:
                                 mimetype=MIMETYPE_PAGE,
                                 content=to_xml(result.pcgts))
 
-    def process_page_pcgts(self, *input_pcgts : OcrdPage, output_file_id : Optional[str] = None, page_id : Optional[str] = None) -> OcrdPageResult:
+    def process_page_pcgts(self, *input_pcgts : Optional[OcrdPage], page_id : Optional[str] = None) -> OcrdPageResult:
         """
         Process the given ``input_pcgts`` of the :py:attr:`workspace`,
         representing one physical page (passed as one parsed
         :py:class:`~ocrd_models.OcrdPage` per input fileGrp)
         under the given :py:attr:`parameter`, and return the
-        resulting :py:class:`~ocrd_models.OcrdPage`.
-
-        Optionally, return a list or tuple of the :py:class:`~ocrd_models.OcrdPage`
-        and one or more lists or tuples of :py:class:`PIL.Image` (image data),
-        :py:class:str (file ID) and :py:class:str (file path) of derived images
-        to be annotated along with the resulting PAGE file.
+        resulting :py:class:`~ocrd.processor.ocrd_page_result.OcrdPageResult`.
+
+        Optionally, add to the ``images`` attribute of the resulting
+        :py:class:`~ocrd.processor.ocrd_page_result.OcrdPageResult` instances
+        of :py:class:`~ocrd.processor.ocrd_page_result.OcrdPageResultImage`,
```

Comment:
I would suggest exporting this on `ocrd.processor` module level for simplicity.

```suggestion
        resulting :py:class:`~ocrd.processor.OcrdPageResult`.

        Optionally, add to the ``images`` attribute of the resulting
        :py:class:`~ocrd.processor.OcrdPageResult` instances
        of :py:class:`~ocrd.processor.OcrdPageResultImage`,
```

Reply:
done 5718cf92 f5f3145e